src/repos/player.js

```diff
@@ -0,0 +1,148 @@
+var sql = require('pg-sql').sql
```

Comment:
Funny, I adopted the more vertical style because it's supposed to be more readable (things line up better, indented blocks imply isolated contexts just like in a standard language). I'd rather stick with this simply because I don't see much value in switching what's existing.